### DISCUSSION TITLE
Is there any way to access the Redis client?

### DISCUSSION THREAD

**Discussion by danechitoaie** [litestar-org/litestar#1338] [Q&A] [answered]
I'm using cache with Redis backend but we only have get/set/delete methods. Is there any way to run different commands? I would like to run a count so I get the number of all items from the cache with a certain prefix for the cache key so I can count the number of online users.

**Comment by provinzkraut** [comment, marked as answer]
There is, but not through a public API unfortunately.

You can use `cache.backend._redis` to get the client used.

Check out #1330, which proposes a new API for this, that would greatly simplify this access.

**Comment by provinzkraut** [comment]
Implemented in #1330
